app/assets/javascripts/shops/order.js

```diff
@@ -0,0 +1,25 @@
+$(function() {
+  $('#btn-update-order').click(function() {
+    let status = $(this).parent().find('#order-status').val();
```

Comment:
đặt 1 biến cho cái này
$(this).parent().find('#order-status') rồi tái sử dụng lại e

Reply:
dạ vâng ạ